### PR TITLE
docs: Fix s1ap federated integ test (missing build)

### PR DESCRIPTION
**Pull request by uri200**
Signed-off-by: Oriol Batalla <obatalla@fb.com>

## Summary
Add a note to build the images the first time we run federated integ test

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: docs/docusaurus/versioned_docs/version-1.7.0/feg/s1ap_federated_test.md [hunk 1]
```diff
@@ -57,6 +57,10 @@ This script will
 
 ```bash
 cd magma/lte/gatewat
```
Comment (christinewang5) [inline docs/docusaurus/versioned_docs/version-1.7.0/feg/s1ap_federated_test.md:59]: nit typo here...

**Review by christinewang5 — APPROVED** [review]
lgtm! thanks for fixing

File: docs/docusaurus/versioned_docs/version-1.7.0/feg/s1ap_federated_test.md (hunk 1 above)
Comment (Neudrino) [reply docs/docusaurus/versioned_docs/version-1.7.0/feg/s1ap_federated_test.md:59]: Will be fixed in https://github.com/magma/magma/pull/12610.

**Review by Neudrino — APPROVED** [review]

**Comment by Neudrino** [comment]
[Sematic PR](https://github.com/magma/magma/blob/master/.github/workflows/semantic-pr.yml) needs to be fixed here, @uri200. E.g. `docs:` or the like.

**Review by voisey — APPROVED** [review]

**Pull request merged**